src/utrfx/variant_util.py

```diff
@@ -129,3 +167,25 @@ def retrieve_variants_of_region(
             for alt in rec.alts:
                 variant_list.append(VariantCoordinates.from_vcf_literal(contig=contig, pos=pos, ref=ref, alt=alt))
         return variant_list
+    
+    def get_allele_frequency(
```

Comment:
What happens if two alt alleles are at one location? This is not uncommon.

Three ALT alleles can exist for a single position. E.g. for `X:100-101:A`, the following can happen
* `X:100-101:A>C`
* `X:100-101:A>G`
* `X:100-101:A>T`

and we don't even consider INDELs or MNVs, such as
* `X:100-102:AA>A`
* `X:100-102:AA>CT`
* ...

So, the signature of the `get_allele_frequency` method should take `VariantCoordinates` instead of `contig`, `start`, and `end`. Returning optional `float` sounds OK.